.github/workflows/release.yml

```diff
@@ -1,86 +1,100 @@
 name: Release version
+
 on:
   push:
     tags:
       - 'v*.*.*'
+  schedule:
+    - cron: '0 0 * * *'
+
+env:
+  TAG_NAME: ${{ github.event_name == 'schedule' && 'nightly' || github.ref_name }}
```

Comment:
I've tested the main release method for hours (had issues with Windows) but I have not been able to test the nightly flow. I'm not sure how to

Reply:
WDYM "main" release vs "nightly" flow?

Reply:
I've only tested tag releases, not the scheduled one. I couldn't figure out how to trigger it on my fork

Reply:
macos-latest runner is intel only fyi

Reply:
yeah we're having issues compiling to M1 so we're not supporting that for now :(